packages/apps/shopify-app-remix/src/server/authenticate/public/checkout/authenticate.public.checkout.doc.ts

```diff
@@ -35,7 +35,14 @@ const data: ReferenceEntityTemplateSchema = {
     },
   },
   jsDocTypeExamples: ['CheckoutContext'],
-  related: [],
+  related: [
```

Comment:
Totally a nit, so feel free to ignore: usually in these related links, I try to use the Shopify icon instead of the standard one so they stand out a bit more.